elastic/kibana
# [Uptime] adjust tls defaults for monitor edit flow

## Summary

Resolves https://github.com/elastic/kibana/issues/150949

[Add-Monitor-Uptime---Kibana.webm](https://user-images.githubusercontent.com/11356435/218583413-8b321365-ad89-4b4b-bb13-7fe26c6a18dd.webm)


### Testing
1. Create a TCP monitor in the Uptime with the following host 8.8.8.8:80
2. Navigate to the monitor details page for in Uptime app.
3. Confirm that the url for the monitor is tcp://8.8.8.8:80, not ssl:/8.8.8.8:80.
4. Navigate back to edit page for Uptime
5. Confirm that the tls button is not toggled to on
6. Re-save the monitor.
7. Wait for the next monitor run. Confirm the url for the monitor is tcp://8.8.8.8:80, not ssl:/8.8.8.8:80.
